### PR TITLE
Remove linear erc20 voting

### PR DESCRIPTION
**Pull request by adamgall**
Please review and merge https://github.com/decentdao/fractal-interface/pull/1666 first.

### CODE REVIEW

**Review by Da-Colon — APPROVED** [review]
No comments. Approved

**Comment by Da-Colon** [comment]
image

needs to make up its mind.

**Comment by adamgall** [comment]
Whoops! not sure how that happened. Fixing now.

**Comment by adamgall** [comment]
> @adamgall Apologies, I merged and deleted the branch that was targeted in this PR and seems like now we can't re-open this PR - do you mind opening up another one targeting root remove typechain branch and reflecting on comments that I left in the review?

@mudrila i was able to open this one back up after some branch manipulation 👍

File: src/hooks/DAO/loaders/governance/useAzoriusListeners.ts [hunk 1]
```diff
@@ -229,15 +210,38 @@ export const useAzoriusListeners = () => {
...
     }
 
-    const votedEvent = erc20StrategyContract.filters.Voted();
-    const listener = erc20VotedEventListener(erc20StrategyContract, strategyType, action.dispatch);
-
-    erc20StrategyContract.on(votedEvent, listener);
+    const unwatch = erc20StrategyContract.watchEvent.Voted({
+      onLogs: async logs => {
+        for (const log of logs) {
+          if (!log.args.proposalId || !log.args.voter || !log.args.voteType || !log.args.weight) {
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/hooks/DAO/loaders/governance/useAzoriusListeners.ts:216]: @mudrila I guess I'm not sure what you mean by this, they're arriving "non-parsed" and "we need to parse them". can you point me at some docs describing what you mean by this?

**Pull request merged**